test/__init__.py

```diff
@@ -1,17 +1,8 @@
-import inspect
 import os
 from contextlib import contextmanager
+from pathlib import Path
```

Comment:
Thanks for adding this fix as well! I incorporated this into a different PR, so it may or may not show up if that one does/does not get added first, but I used your code here to do it.